tests/src/test/java/alluxio/client/fs/io/FileInStreamIntegrationTest.java

```diff
@@ -101,7 +101,8 @@ public FileInStreamIntegrationTest(BlockStoreType blockStoreType) {
           .setProperty(PropertyKey.USER_STREAMING_READER_CHUNK_SIZE_BYTES, Constants.KB)
           .setProperty(PropertyKey.USER_CLIENT_CACHE_SIZE, ImmutableList.of(100 * Constants.MB))
           .setProperty(PropertyKey.USER_CLIENT_CACHE_DIRS,
-              ImmutableList.of(AlluxioTestDirectory.ALLUXIO_TEST_DIRECTORY));
+              ImmutableList.of(AlluxioTestDirectory.createTemporaryDirectory("page_store")
+                  .getAbsolutePath()));
```

Comment:
`AlluxioTestDirectory.ALLUXIO_TEST_DIRECTORY` is static constant, so for every test it's the same directory, this will cause interference between tests.